docs/src/modules/components/AppFrame.js

```diff
@@ -175,79 +181,81 @@ export default function AppFrame(props) {
   const openDrawer = React.useCallback(() => setMobileOpen(true), []);
 
   const { activePage, productIdentifier } = React.useContext(PageContext);
+  const themeOptions = React.useContext(ThemeOptionsContext);
 
   const disablePermanent = activePage?.disableDrawer === true || disableDrawer === true;
 
   return (
-    <RootDiv className={className}>
-      <NextNProgressBar />
-      <CssBaseline />
-      <SkipLink />
-      <MarkdownLinks />
-      <StyledAppBar
-        disablePermanent={disablePermanent}
-        sx={{ minHeight: 'var(--MuiDocs-header-height)' }}
-      >
-        <GlobalStyles
-          styles={{
-            ':root': {
-              '--MuiDocs-header-height': `${HEIGHT}px`,
-            },
-          }}
-        />
-        <Stack direction="row" sx={{ alignItems: 'center', position: 'relative', width: '100%' }}>
-          <NavIconButton
-            edge="start"
-            color="primary"
-            size="small"
-            aria-label={t('appFrame.openDrawer')}
-            disablePermanent={disablePermanent}
-            onClick={() => setMobileOpen(true)}
-            sx={{ ml: '1px' }}
-          >
-            <SvgHamburgerMenu />
-          </NavIconButton>
-          <Box sx={{ display: { md: 'flex', lg: 'none' } }}>
-            <LogoWithCopyMenu
-              logo={productIdentifier.logo}
-              logoSvgString={productIdentifier.logoSvg}
-              wordmarkSvgString={productIdentifier.wordmarkSvg}
-              marginLeft
-            />
-          </Box>
-          <Stack direction="row" spacing={1} useFlexGap sx={{ ml: 'auto' }}>
-            <BannerComponent />
-            <DeferredAppSearch />
-            <Tooltip title={t('appFrame.github')} enterDelay={300}>
-              <IconButton
-                component="a"
-                color="primary"
-                size="small"
-                href={process.env.SOURCE_CODE_REPO}
-                data-ga-event-category="header"
-                data-ga-event-action="github"
-              >
-                <GitHubIcon fontSize="small" />
-              </IconButton>
-            </Tooltip>
-            <Notifications />
-            <Tooltip title={t('appFrame.toggleSettings')} enterDelay={300}>
-              <IconButton color="primary" size="small" onClick={() => setSettingsOpen(true)}>
-                <SettingsIcon fontSize="small" />
-              </IconButton>
-            </Tooltip>
+    <BrandingCssVarsProvider {...themeOptions}>
```

Comment:
Have you checked if there are other places where this needs to be done? This AppFrame as far as I know is used only on subset of the pages. (I may be wrong with this assumption)

Reply:
Yes, I checked. AppFrame only used on demo pages.

Reply:
So it's the plan to add this on the other pages as well directly?

Reply:
They are all added. Marketing, blog, and API pages already contain BrandingCssVarsProvider.

Might need to adjust the API pages a bit because it's also using AppFrame.